modules/organization-policy/README.md

```diff
@@ -0,0 +1,166 @@
+# Google Cloud Organization Policy
+
+This module allows creation and management of [GCP Organization Policies](https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints) by defining them in a well formatted `yaml` files or with HCL.
+
+Yaml based factory can simplify centralized management of Org Policies for a DevSecOps team by providing a simple way to define/structure policies and exclusions.
+
+## Example
+
+### Terraform code
+
+```hcl
+# using configuration provided in a set of yaml files
+module "org-policy-factory" {
+  source = "./modules/organization-policy"
+
+  config_directory = "./policies"
+}
+
+# using configuration provided in the module variable
+module "org-policy" {
+  source = "./modules/organization-policy"
+  
+  organization_policies = {
```

Comment:
I don't much like having an `organization_policies` variable in a module that only deals with organization policies ;)

Reply:
I will go for `policies`